server/bot/poster.go

```diff
@@ -159,6 +159,10 @@ func (b *Bot) NotifyAdmins(messageType, authorUserID string, isTeamEdition bool)
 		message = fmt.Sprintf("@%s requested access to the timeline in Incident Collaboration.", author.Username)
 		title = "Keep a complete record of the playbook run timeline"
 		text = "The playbook run timeline automatically tracks key events and messages in chronological order so that they can be traced and reviewed afterwards. Teams use timeline to perform retrospectives and extract lessons for the next time that they run the playbook."
+	case "start_trial_to_access_playbook_dashboard":
+		message = fmt.Sprintf("@%s requested access to the playbook dashboard in Incident Collaboration.", author.Username)
```

Comment:
We are removing the "in Incident Collaboration" part. See https://github.com/mattermost/mattermost-plugin-incident-collaboration/pull/611/files
```suggestion
		message = fmt.Sprintf("@%s requested access to the playbook dashboard.", author.Username)
```